src/webplugin/popup.ts

```diff
@@ -0,0 +1,155 @@
+let dbName = "_ionicstorage";
+
+let dbOpenRequest = indexedDB.open(dbName);
+
+let baseUrl = "";
+
+if (isDevMode()) {
+    baseUrl = "http://localhost:8090";
+} else {
+    baseUrl = "https://prototype-api.smilo.network";
+}
+
+let browser = (<any>window).browser = (function () {
+    return (<any>window).msBrowser ||
+    (<any>window).browser ||
+    (<any>window).chrome;
+})();
+
+let totalWallets = [];
+let totalAssets = [];
+let currentPublicKey;
+
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
Didn't we include JQuery in this project? If so might make more sense to use JQuery for all DOM selection queries and event handling. E.g. all `addEventListener` calls and all `getElementBy...` calls.